Jewish Calendar: configuration via config flow (#11176)
home-assistant/home-assistant.github.io: source/_integrations/jewish_calendar.markdown

## Configuration

Set up the integration through **Configuration** -> **Integrations** -> **Jewish Calendar**.
For legacy support the old Jewish Calendar configuration is imported and set up as a new integration.

> To enable this integration in your installation, add the following to your `configuration.yaml` file:

Reviewer: Consider changing this wording. It now feels conflicted with the second above.

```suggestion
Alternatively, you can enable this integration by adding the following to your `configuration.yaml` file:
```
